src/Splat/Platforms/Android/Bitmaps/PlatformBitmapLoader.cs

```diff
@@ -108,5 +96,36 @@ public IBitmap Create(float width, float height)
         {
             return Bitmap.CreateBitmap((int)width, (int)height, Bitmap.Config.Argb8888).FromNative();
         }
+
+        internal static Dictionary<string, int> GetDrawableList()
+        {
+            var log = Splat.LogHost.Default;
```

Comment:
Should we return a ImmutableDictionary? It's thread safe.

debatable in this case since we only ever read from it.